.gitignore

```diff
@@ -18,3 +18,4 @@ flake8.report
 junit*.xml
 doc/build
 .cache
+.idea
```

Comment:
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Thanks for opening this up once again! Having windows support for testinfra is a really nice feature. Is it planned to release the feature soon? Also, is there any workaround till then? Any comment or suggestion is appreciated!